cartography/intel/gcp/__init__.py

```diff
@@ -194,7 +230,13 @@ def _sync_single_project(
     if service_names.dns in enabled_services:
         dns.sync(neo4j_session, resources.dns, project_id, gcp_update_tag, common_job_parameters)
     if service_names.cloudfunction in enabled_services:
-        cloudfunction.sync(neo4j_session, resources.function, project_id, gcp_update_tag,common_job_parameters)
+        cloudfunction.sync(neo4j_session, resources.cloudfunction, project_id, gcp_update_tag, common_job_parameters)
+    if service_names.cloudsql in enabled_services:
```

Comment:
I see your code structure here. I misspoke earlier. Can we break databases.py into 3 separate files?
1. cloudsql
2. bigtable
3. firestore